udaykalra/portfolio/src/main/webapp/style.css

```diff
@@ -144,4 +149,5 @@ body {
   background-color: var(--body-color);
   height: 100%;
   margin: 0;
-}
\ No newline at end of file
+}
+>>>>>>> week-3-one
```

Comment:
This looks like a conflict marker that needs to be removed.

Reply:
Done